packages/cms-cli/commands/init.js

```diff
@@ -59,7 +60,18 @@ function initializeConfigCommand(program) {
 
       try {
         const configData = await personalAccessKeyPrompt();
-        await updateConfigWithPersonalAccessKey(configData, true);
+        const { name } = await promptUser([PORTAL_NAME]);
+        await updateConfigWithPersonalAccessKey(
+          {
+            ...configData,
+            name,
+          },
+          true
+        );
+
+        logger.log(
+          `Success: ${DEFAULT_HUBSPOT_CONFIG_YAML_FILE_NAME} created with ${PERSONAL_ACCESS_KEY_AUTH_METHOD.name}.`
+        );
```

Comment:
Not something for this PR, but I think we should consider adding `logger.success` and having a `[Success]` prefix that we color green.

Reply:
> Not something for this PR, but I think we should consider adding logger.success and having a [Success] prefix that we color green.

@gcorne This is for the console, correct? Would the plan be to duplicate the message or provide additional info? I was thinking about this in the context of @HubSpotProtected/LocalDevAuth#31 where we have both an error and info. Should I be thinking about console logs in the same vein as I'm thinking about standardized terminal messages and prompts (e.g., ensuring consistency across scenarios)?

Reply:
The idea is that in a similar fashion to errors that we have a system for displaying success messages in a somewhat similar fashion to how it works in Canvas.

The following is an example of an error:
![image](https://user-images.githubusercontent.com/266036/75904277-e6a0b180-5e10-11ea-804d-fff801e4755b.png)

Reply:
Fixed in https://github.com/HubSpot/hubspot-cms-tools/pull/125